src/components/atoms/links/Link.tsx

```diff
@@ -40,6 +51,7 @@ const Link: React.FC<LinkProps> = ({
       rel="noreferrer"
       className={className}
       onKeyUp={handleKeyUp}
+      onClick={handleClick}
```

Comment:
It seems like there are two things going on in this PR:

1. Adding support for tracking clicks/activation of links (added here)
2. Tracking recommended materials

It would be nice to have this in separate commits. Keep this in mind the next time around :smile:

Reply:
Yes, the exact same code for the Link component was added in PR 1764 https://github.com/danskernesdigitalebibliotek/dpl-react/pull/1764/commits/6bb9eb5b11f87b00d180c12993e9519d6873e42a

But yes, it should have been in a separate commit so that I could have cherry-picked it. 😄